packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -66,38 +70,86 @@ export default {
       default: false,
     },
     /**
-     * Validate value of form select
+     * Hint/Required message value of select.
      */
-    valid: {
-      type: Boolean,
-      default: true,
+    hintMessage: {
+      type: String,
+      default: "Required.",
     },
     /**
-     * Disabled status of form select
+     * Validate value of select
      */
-    disabled: {
+    valid: {
       type: Boolean,
-      default: false,
+      default: true,
     },
     /**
-     * Error message value of form select. It will be appeared if `valid` is `true`.
+     * Error message value of select. It will be appeared if `valid` is `true`.
      */
     errorMessage: {
       type: String,
       default: "This field is not correct.",
     },
     /**
-     * Adds placeholder
+     * Info/success message value of select.
+     */
+    infoMessage: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Disabled status of form select.
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * Adds placeholder.
      */
     placeholder: {
       type: String,
       default: "",
     },
   },
-  data() {
-    return {
-      selectedValue: "",
```

Comment:
Why is it deleted? :) there are errors in the console regarding this value

Reply:
You are right, I restored selectedValue.